paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -881,6 +883,8 @@ class PaymentSheet internal constructor(
     }
 
     @Parcelize
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
We don't want to restrict all of appearance! That would be bad! I think you meant to restrict the constructor.